docs/dev-guides/tutorials/references/prepare-datahub.md

```diff
@@ -0,0 +1,38 @@
+# Prepare Local Datahub Environment
+
+## Deploy Datahub Quickstart 
+
+You need a datahub running in local environment. First, install acryl-datahub if you did't yet. 
+```shell
+python3 -m pip install --upgrade pip wheel setuptools
+python3 -m pip install --upgrade acryl-datahub
+```
+If you can see datahub version like this, you're good to go. 
+```shell
+$ datahub version
+DataHub CLI version: 0.10.0.1
+Python version: 3.9.6 (default, Jun 16 2022, 21:38:53)
+[Clang 13.0.0 (clang-1300.0.27.3)]
```

Comment:
Yep, literally the same. At first, I had the quickstart instructions on the same page, but then I separated them because they seemed repetitive. But now It can be seen as less coherent.
@maggiehays What do you think about this?